packages/core/integration-tests/test/eslint-validation.js

```diff
@@ -0,0 +1,40 @@
+import assert from 'assert';
+import path from 'path';
+import {bundle} from '@parcel/test-utils';
+import defaultConfigContents from '@parcel/config-default';
+
+const config = {
+  ...defaultConfigContents,
+  validators: {
+    '*.{js,jsx,ts,tsx}': ['@parcel/validator-eslint']
+  },
+  reporters: [],
+  filePath: require.resolve('@parcel/config-default')
+};
+
+// For some reason this throws uncaught after passing...
```

Comment:
promise not being awaited?

Reply:
Just figured it out, apparently the promiseQueue.add also returns the result and therefore also error... which we don't handle for validation.

Added a no-op catch and it's fixed :) Seems like it's been there since I wrote validators....